src/components/ShareURIDialog.vue

```diff
@@ -32,7 +40,9 @@
 <script>
```

Comment:
After fixing the issues, I suggest you to do a little refactor by rewriting the component using Composition API and TypeScript. Follow the guidelines https://developers.adamant.im/template-classes.html